releases.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Releases
+layout: default
```

Comment:
Similar to the News page, we should use a consistent layout template (like the one used for the other pages like [about](https://github.com/sjsrey/pysal.github.io/blob/jekyll/about.md) and [Documentation](https://github.com/sjsrey/pysal.github.io/blob/jekyll/documentation.md)). Therefore, it should be `layout:page_md`.

Reply:
will fix. good catch.